main.go

```diff
@@ -102,6 +102,12 @@ func main() {
 			Destination: &config.NotifyInterval,
 			Value:       time.Minute * 10,
 		},
+		&cli.StringFlag{
+			Name:        "emulated-etcd-version",
+			Usage:       "The emulated etcd version to return on a call to the status endpoint. Defaults to 3.4.31 which is the least version that support Kubernetes watch progress feature.",
+			Destination: &config.EmulatedETCDVersion,
+			Value:       "3.4.31",
```

Comment:
I'd prefer to claim to be 3.5.x, as that is the version of etcd code I referenced when adding WatchProgressNotify support to kine. 
```suggestion
			Usage:       "The emulated etcd version to return on a call to the status endpoint.  Defaults to 3.5.13, in order to indicate support for watch progress notifications.",
			Destination: &config.EmulatedETCDVersion,
			Value:       "3.5.13",
```